pom.xml

```diff
@@ -302,5 +304,16 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>windows</id>
+      <activation>
+        <os>
+          <family>windows</family>
+        </os>
+      </activation>
+      <properties>
+        <os.windows>true</os.windows>
```

Comment:
Why not simply declaring `<cyclonedx.skip>true</cyclonedx.skip>` here? IMO it makes it easier to understand what this profile is about.